.github/workflows/compilation.yml

```diff
@@ -263,6 +267,11 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
+      with:
+        fetch-depth: '0'
+
+    - name: Set OPENOMF_VERSION
+      run: echo "OPENOMF_VERSION=$(git describe --tags --match '[0-9]*.[0-9]*.[0-9]*')" >> "$GITHUB_ENV"
```

Comment:
(if we're keeping git describe)
Could you leave a comment stating that `--match` does not take a regex, but rather a glob(7)?
https://git-scm.com/docs/git-describe#Documentation/git-describe.txt---matchltpatterngt
https://linux.die.net/man/7/glob

could use a glob that looks less like a regex, too, like `--match '[[:digit:]]*'; this glob matches any tag starting with a digit, and omits the check for periods for brevity.